src/runtime/refiner.ts

```diff
@@ -129,6 +129,7 @@ export class Refinement {
       const rangeB = Range.fromExpression(b.expression);
       return rangeA.isSubsetOf(rangeB) ? AtleastAsSpecific.YES : AtleastAsSpecific.NO;
     } catch (e) {
+      console.warn(`Unable to ascertain if ${a} is at least as specific as ${b}.`);
```

Comment:
For the record: this should (ideally) be put into a warnings field in the `options` arg which should be plumbed through all our type checking.

Reply:
Ah right.. is that something I should look at as part of this PR?

Reply:
Nope, just a note for later viewing.